ts/components/wallet/card/Logo.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import {
+  CreditCard,
+  CreditCardType,
+  getCardType
+} from "../../../types/CreditCard";
+
+export enum LogoPosition {
+  TOP,
+  CENTER
+}
+
+export const shouldRenderLogo = (
+  current: LogoPosition,
+  requested?: LogoPosition
+) => {
+  return current === requested; // current position is the requested one
+};
+
+export const getCardIcon = (cc: CreditCard) => {
+  const type = getCardType(cc);
+  const icons: { [key in CreditCardType]: any } = {
+    MASTERCARD: require("../../../../img/wallet/cards-icons/mastercard.png"),
```

Comment:
@matteodesanti can we have vector versions of these logos? I'm sure they can be found online

Reply:
here they are :)
[credit-cards-icons.zip](https://github.com/teamdigitale/italia-app/files/2106119/credit-cards-icons.zip)